docs/apache-airflow-providers-slack/index.rst

```diff
@@ -44,7 +44,7 @@ Content
 
     Connection Types <connections/slack>
     Python API <_api/airflow/providers/slack/index>
-    Example DAGs <https://github.com/apache/airflow/tree/providers-slack/5.0.0/tests/system/providers/slack>
```

Comment:
I think that  was fine. All the other providers use it and we are (I believe automatically updating those).